server/lib/config/pbench-server-default.cfg

```diff
@@ -197,12 +197,6 @@ user = %(default-user)s
 mailfrom = %(user)s@%(host)s
 tasks = pbench-audit-server
 
-[pbench-prep]
-host = %(default-host)s
-user = %(default-user)s
-mailfrom = %(user)s@%(host)s
-tasks = pbench-prep-task
-
```

Comment:
Maybe best to defer this for a later PR: it breaks a number of legacy tests, and cleaning that up would expand and diffuse the primary focus.